extensions/toggl-track/src/index.tsx

```diff
@@ -25,6 +25,21 @@ function ListView() {
       [] as TimeEntry[]
     );
 
+  const totalDurationToday =
+    timeEntries
+      .filter((timeEntry) => dayjs(timeEntry.start).isSame(dayjs(), "day"))
```

Comment:
This is a tricky one, isn't it? Current approach can show today's duration longer than today is, e.g. a couple of hours at 00:01.

I'd say it's good enough, and the complexity of splitting it at midnight might be not worth the trouble.

Reply:
Actually the running entry is always added, even if didn't start today 🤔 Maybe would be better to also check for the start date there as well, what do you think?

Reply:
I think it depends on the specific use case but both approaches are fine.